samples/scenarios/jdbc_kerberos/README.md

```diff
@@ -0,0 +1,152 @@
+
+# Readme
+
+The postgres jdbc kerberos sample connector plugin demonstrates how to incorporate different kerberos-based authentication modes for your "Tableau + Postgres + JDBC" use cases. This sample plugin code has been tested for all the authentication methods listed here with all platforms (Mac Desktop, Windows Desktop and Server, Linux Server). However, the individual sections of the plugin components concern specific authentication modes and have been laid out here for the purposes of understanding and re-use. While publishing from Tableau Desktop, the Server RunAs and Viewer credentials authentication options will be visible in publish dialog only if you have logged into tableau using Integrated Authentication. 
```

Comment:
Not a big deal, but next time try to use consistent capitalization for public facing stuff like "Postgres JDBC"